www/index.md

```diff
@@ -4,32 +4,39 @@ title: ///_hyperscript
 ---
 
 <div style="background-color: lightgoldenrodyellow; margin: 16px; border-radius: 8px;
-            color: darkgoldenrod; border: gold 1px solid; font-size: 20px">
-  <p>
-    <b>NOTE:</b> hyperscript is still in very early development and may change
-    dramatically between releases.  Please bear this in mind, and jump on the
-    <a style="color: darkgoldenrod;font-weight: bold" href="https://htmx.org/discord">#hyperscript discord channel</a> to work with us as we develop the language.  Thank you!
-  </p> 
+color: darkgoldenrod; border: gold 1px solid; font-size: 20px">
+<p>
+hyperscript is in active development and is working to a 1.0 release.
+At this time the syntax and core feature set are considered to be reasonably complete.
+Key areas of focus for 1.0 include additional test cases and documentation improvements.
+Please join us at the 
+<a style="color: darkgoldenrod;font-weight: bold" href="https://htmx.org/discord">#hyperscript discord channel</a>
+to work with us as we push to a public release.  Thank you!
+</p> 
 </div>
 
 ## intro
 
-hyperscript is an experimental scripting language designed for the web, inspired by 
- [HyperTalk](https://hypercard.org/HyperTalk%20Reference%202.4.pdf)
+hyperscript is a scripting language designed for modern front-end web development. 
```

Comment:
I'd imagine a lot of the target audience for hyperscript is put off by "modern" web development. Maybe "easy" or "approachable"?

Reply:
I used modern because dynamic was worse, hah.

I like "easy and approachable" a lot.

LMK if you would like me to submit PR or if @1cg just want to commit.